lib/workers/repository/stats.ts

```diff
@@ -30,25 +30,21 @@ export function printRequestStats(): void {
     requestHosts[hostname].push(httpRequest);
   }
   logger.trace({ allRequests, requestHosts }, 'full stats');
-  const hostStats: string[] = [];
+  const hostStats: Record<string, any> = {};
```

Comment:
To lazy for a real type def? 😉 
```suggestion
  type HostStats = { requestCount: number, requestAvgMs: number, queueAvgMs: number };
  const hostStats: Record<string, HostStats > = {};
```